cmd/pebble/cmd_plan.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"github.com/jessevdk/go-flags"
+
+	"github.com/canonical/pebble/client"
+)
+
+type cmdPlan struct {
+	clientMixin
+}
+
+var shortPlanHelp = "Show the plan with layers combined"
+var longPlanHelp = `
+The plan command reads the plan (configuration) and displays it as YAML. The
+plan's layers are combined according to Pebble's layer override rules.
```

Comment:
It feels like there's just a bit of tension between the concepts described here.

I've tried to take that same idea and create something similar: 
```
The plan command prints out the complete configuration of pebble as it stands
in memory. Layers are combined according to the override rules defined in them.
``` 
How do you like it?

Reply:
I can see what you're saying. However, I don't like "as it stands in memory" -- that's not very intuitive not correct, because in memory (taken literally) it's laid out however the Go struct/map internals are laid out. It's a YAML encoding, so I'd rather explicitly say that. I've tweaked to a (hopefully good enough) combination, including the word "effective" which might capture what you mean.

Reply:
That's great as well, thanks.